addOns/spider/src/main/javahelp/org/zaproxy/addon/spider/resources/help/contents/options.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 3.2 Final//EN">
+<HTML>
+<HEAD>
+<META HTTP-EQUIV="Content-Type" CONTENT="text/html; charset=utf-8">
+<TITLE>
+Spider dialog
```

Comment:
Just noticed the contents don't match the file name, that might be cause of the links not working. This one and `tab.html` seem to have the same content as `dialog.html`

Reply:
Crap, might be a bad copy/paste.....grrr. TY

Reply:
Thanks that was it. Looks like I need copies of images for the tab page too.